Source/SwiftLintFramework/Extensions/NSFileManager+SwiftLint.swift

```diff
@@ -21,7 +21,7 @@ extension FileManager: LintableFileManager {
             .standardizingPath
 
         // if path is a file, it won't be returned in `enumerator(atPath:)`
-        if absolutePath.bridge().isSwiftFile() && absolutePath.isFile {
+        if absolutePath.isFile {
```

Comment:
why this change? I think the `lintablePaths` change is sufficient to support the `lint --path file.notswift` scenario.